docs/UserGuide.md

```diff
@@ -722,7 +788,7 @@ Action | Format, Examples
 **Find Appointment** | `findappt KEYWORD [MORE_KEYWORDS] [by/(person OR name)]` <br> e.g `findappt Meeting`
 **Help** | `help`
 **Export CSV** | `exportcsv`
-**Import CSV** | `importcsv f/FILEPATH [n/COLUMNNUM] [p/COLUMN_PERSON] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]` <br> e.g., `importCSV n/2 p/3 e/5 a/6 t/4`
+**Import CSV** | `importcsv f/FILEPATH [n/COLUMNNUM] [p/COLUMN_PERSON] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]` <br> e.g., `importcsv n/2 p/3 e/5 a/6 t/4`
```

Comment:
my bad on this but can help me change this to
```suggestion
**Import CSV** | `importcsv f/FILEPATH [n/COLUMNNUM] [p/COLUMN_PERSON] [e/COLUMN_EMAIL] [a/COLUMN_ADDRESS] [t/COLUMN_TAGS]` <br> e.g., `importcsv f/data/csvFile.csv n/2 p/3 e/5 a/6 t/4`
```

Reply:
Adopted in 40d26e910ce1b3157f5e11b3fc9b2bcc0c5092de.